### PR TITLE
In 150680241 contact method and location

### PR DESCRIPTION
**Pull request by saraOSI**
### Pivotal Story

- [Investigator can record where and how a contact occurred #150680241](https://www.pivotaltracker.com/story/show/150680241)

### Purpose
Add LOVs for contact location and update the display value of communication method LOVs to match what is displayed on a screening.

### Testing Notes
I've uped and downed both of these migrations locally to make sure the rollback is in working order.

### CODE REVIEW

File: db/migrate/20170925172154_rename_communication_method_values.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+  def down
+    category_id = execute(<<-SQL).first['id']
+      SELECT id FROM intake_lov_categories
+      WHERE intake_type = 'communication_method'
+    SQL
+
+    execute(<<-SQL)
+      UPDATE intake_lov_categories
+      SET intake_type = 'COMMUNICATION_METHOD'
```
Comment (NateOSI) [inline db/migrate/20170925172154_rename_communication_method_values.rb]: hasn't the down casing of 'COMMUNICATION_METHOD' already been done by the migration I wrote last week?

Comment (saraOSI) [reply db/migrate/20170925172154_rename_communication_method_values.rb]: Nope. It wasn't one of the values you grabbed.

File: db/migrate/20170922163011_add_location_lovs.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      (#{category_id}, 415, 415, 'CWS Office', 'f', null),
+      (#{category_id}, 417, 417, 'Home', 'f', null),
+      (#{category_id}, 5524, 5524, 'In placement', 'f', null),
+      (#{category_id}, 420, 420, 'School', 'f', null),
+      (#{category_id}, 418, 418, 'Other', 'f', null);
+    SQL
+  end
+
+  def down
+    category_id = execute(<<-SQL).first['id']
```
Comment (ThomasOSI) [inline db/migrate/20170922163011_add_location_lovs.rb]: Not expecting this to be a big risk but searching for distinct would help ensure no contact_location records become orphans

File: db/migrate/20170925172154_rename_communication_method_values.rb [hunk 3]
```diff
@@ -0,0 +1,50 @@
+class RenameCommunicationMethodValues < ActiveRecord::Migration[5.1]
+  COMMUNICATION_METHODS = [
```
Comment (ThomasOSI) [inline db/migrate/20170925172154_rename_communication_method_values.rb]: Missing "Fax"?

Comment (saraOSI) [reply db/migrate/20170925172154_rename_communication_method_values.rb]: Fax is already properly named, so we don't need to update it.

**Pull request merged**